myapp/app/controllers/tasks_controller.rb

```diff
@@ -15,11 +15,12 @@ def create
     @task = Task.new(task_params)
 
     if @task.save
-      flash[:success] = 'タスク投稿が成功しました'
+      flash[:success] = I18n.t('flash.new_success')
       redirect_to @task
```

Comment:
これでもいいですが、一緒にしちゃってもいいかもです
```suggestion
      redirect_to @task, flash: { success: I18n.t('flash.new_success') }
```

Reply:
修正しました。